front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -33,7 +62,22 @@ const ScenarioDescription = ({
 
       <div className="scenario-description">
         {scenario.description && (
-          <div className="scenario-details-description">{scenario.description}</div>
+          <div
```

Comment:
Is it a good idea to re-use the same HTML element for the expanded description? It seems a bit weird to me because the expanded description really looks quite different from the unexpanded one.

Reply:
done